packages/xo-server-transport-icinga2/src/index.js

```diff
@@ -0,0 +1,126 @@
+import assert from 'assert'
+import { URL } from 'url'
+
+// =============================================================================
+
+export const configurationSchema = {
+  type: 'object',
+
+  properties: {
+    server: {
+      type: 'string',
+      description: `
+The icinga2 server http/https address.
+
+*If no port is provided in the URL, 5665 will be used.*
+
+Examples:
+- https://icinga2.example.com
+- http://192.168.0.1:1234
+      `.trim(),
+    },
+    user: {
+      type: 'string',
+      description: 'The icinga2 server username',
+    },
+    password: {
+      type: 'string',
+      description: 'The icinga2 server password',
+    },
+    filter: {
+      type: 'string',
+      description: 'The filter to use',
+    },
```

Comment:
Filters are generally used for querying but we are inserting data, what is its purpose?

Related: is it really required?

Reply:
From what I understood from the docs: filter is used also for the report, it specifies on which objects the report is about.

I guess it can be optional since if no filter is provided its all objects.

Maybe @jcharaoui knows more.